webcompat/templates/home-page/issue-wizard-form.html

```diff
@@ -311,7 +311,7 @@ <h3 class="page-heading">Filing a web compatibility issue</h3>
             <button class="button next-step step-10 issue-btn disabled" data-nextstep="11">Confirm</button>
             <button class="button js-remove-upload is-hidden issue-btn red">Delete this image</button>
           </div>
-          <a class="js-remove-upload is-hidden spaced-link" href="#">Upload a different image</a>
```

Comment:
We're not running tests on this version of the form (which will come back to bite us in the near future), but `js-foo` classes are used by tests, e.g., https://github.com/webcompat/webcompat.com/blob/d56cd783366d725bcf2cf560723a590faf41e08f/tests/functional/image-uploads-non-auth.js#L142

Just something to document here for whoever comes back and writes tests in the future.